pycbc/events/significance.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (C) 2022 Gareth Cabourn Davies
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
```

Comment:
Perhaps this bit can be removed going forwards ... it seems to refer to an era when code was distributed by physical mail

Reply:
True, I just copy-pasted the whole thing from other codes